### ISSUE TITLE
TextInput - Suggestions focus state doesn't have enough contrast

### ISSUE THREAD

**Issue by taysea** [grommet/grommet#7550]
In TextInput with suggestions, the focus indicator stays on the input but we're using color to signify which suggestion is active with keyboard. The color in base grommet theme and grommet-theme-hpe doesn't have enough affordance.

We should re-assess the treatment. Related to WCAG 1.4.1, WCAG 2.4.7

Grommet theme

<img width="495" alt="Image" src="https://github.com/user-attachments/assets/ff2d507e-4c93-4856-ad30-24a6d07da3ff" />

Grommet-theme-hpe

<img width="463" alt="Image" src="https://github.com/user-attachments/assets/aa6b83a3-e630-4555-ad73-3aed0073fb67" />

**Comment by jcfilben** [comment]
@taysea do we have a ticket to track making the needed TextInput options styling adjustment to the HPE theme? If not could we create one?
